modules/ossm-threescale-routing.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/threescale-adapter.adoc
+
+[id="ossm-threescale-routing_{context}"]
+= Routing service traffic through the adapter
+To drive traffic for your service through the 3scale adapter, you need to match the rule `destination.labels["service-mesh.3scale.net"] == "true"` that you previously created in the configuration, in the `kind: rule` resource.
```

Comment:
Please reformat this procedure to follow the template.

Reply:
I have created a rudimentary procedure (numbered steps) but the 3scale team owns this content so @dfennessy will have to refine it.